src/lib/mongoBackend/connectionOperations.cpp

```diff
@@ -359,6 +377,7 @@ bool collectionUpdate
   {
     connection->update(col.c_str(), q, doc, upsert);
     releaseMongoConnection(connection);
+    TIME_STAT_MONGO_WRITE_WAIT_STOP();
```

Comment:
Only the last one, the "part-operation" if you like.
That's for the "last" part.
In "acc", everything will be there, of course.

So, to get what you are looking for, right now you would have to do one 'GET /stat' before an operation, then another one after and do the math yourself.

These measurements aren't "intelligent", except for gathering the accumulated times.